source/MRViewer/MRSurfaceContoursWidget.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "MRViewer.h"
+#include "MRViewerEventsListener.h"
+#include "MRMesh/MRMeshFwd.h"
+#include "MRSurfacePointPicker.h"
+#include <unordered_map>
+#include "MRMesh/MRObjectMeshHolder.h"
+
+
+namespace MR
+{
+
+class SurfaceContoursWidget : public MultiListener<
+    MouseDownListener,
+    MouseMoveListener>
```

Comment:
mouseup?
pls add general comment why is it needed

Reply:
mouseup is non need.